velox/dwio/parquet/reader/ParquetReader.cpp

```diff
@@ -319,7 +333,9 @@ std::shared_ptr<const ParquetTypeWithId> ReaderBase::getParquetColumnInfo(
               std::nullopt,
               std::nullopt,
               maxRepeat + 1,
-              maxDefine);
+              maxDefine,
+              isOptional,
+              1);
```

Comment:
why 1 here? Looks to me this isRepeated, which is a bool value, did you mean true?

Reply:
yes 1 is used in the sense its true. Its not repeated, its child is which we skipped